packages/ui/src/components/atoms/Typography.tsx

```diff
@@ -88,6 +89,11 @@ const baseStyle = css<HeadingProps>`
   ${border};
   ${spacing};
   ${font};
+  ${props =>
+    props.padding &&
+    css`
+      padding: ${props.padding};
+    `};
```

Comment:
Spacing utility should be enough to handle this

Reply:
done